src/NuGet.Core/NuGet.Commands/project.json

```diff
@@ -50,6 +50,11 @@
       "frameworkAssemblies": {
         "System.Xml": "",
         "System.Xml.Linq": ""
+      },
+      "compilationOptions": {
+        "define": [
+          "IS_DESKTOP"
+        ]
       }
     },
     "net46": {

```

Comment:
Why would this project have both "net45" and "net46"? I only see 2 non-test projects with "net46":
* NuGet.Core\NuGet.CommandLine.XPlat
* NuGet.Core\NuGet.Commands

Is that done on purpose?